railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -5,11 +5,23 @@
 <% module_namespacing do -%>
 class <%= controller_class_name %>Controller < ApplicationController
   before_action :set_<%= singular_table_name %>, only: [:show, :edit, :update, :destroy]
+  
+  <%- if collection_routing? -%>
+  # GET <%= route_url %>/1..10,17
+  def index
+    @<%= plural_table_name %> = <%= singular_table_name %>_ids ? <%= orm_class.find(class_name, "#{singular_table_name}_ids") %> : <%= orm_class.all(class_name) %>
+  end
 
+  # <%= singular_table_name.capitalize %> ids
+  def <%= singular_table_name %>_ids
```

Comment:
Good job @ujjwalt ! :sunglasses: 

About this line… is this method intended to be public? Or should `post_ids` be a private method, like `set_post`, `post_params` and `posts_params`? Thanks.